pkg/logger/event/event.go

```diff
@@ -17,39 +17,46 @@
 package event
 
 import (
-	public_source_ip "github.com/AccelByte/public-source-ip"
 	"github.com/emicklei/go-restful"
 	"github.com/fatih/structs"
 	"github.com/sirupsen/logrus"
 )
 
 const (
 	eventLogAttribute     = "EventLog"
-	eventType             = "event"
```

Comment:
To support the current version of event log, I think we still need this event_type to be "event"

Reply:
from our wiki, this type should be int.

Reply:
Actually this const is not used for "event_type" field, but it's used for "log_type". For now I think we still don't want to remove it to keep it compatible with existing log parser.

Reply:
added back now.